.github/workflows/tests.yml

```diff
@@ -0,0 +1,51 @@
+name: tests-pipelinedp
+
+on:
+  push:
+    branches: [ $default-branch ]
+  pull_request:
+    branches: [ $default-branch ]
+
+jobs:
+  linting:
+
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: [3.6,3.7, 3.8, 3.9]
```

Comment:
All versions of python are not needed for linting. One version of py version should work.